packages/fether-react/src/App/App.js

```diff
@@ -73,13 +71,11 @@ class App extends Component {
       <div className='window'>
         {status !== STATUS.GOOD && <Overlay />}
         <Switch>
-          {/* The next line is the hoempage */}
-          <Redirect exact from='/' to='/tokens' />
+          {/* The next line is the homepage */}
+          <Redirect exact from='/' to='/accounts' />
           <Route path='/accounts' component={Accounts} />
           <Route path='/onboarding' component={Onboarding} />
-          <Route path='/send' component={Send} />
-          <Route path='/whitelist' component={Whitelist} />
-          <Route path='/tokens' component={Tokens} />
+          <Route path='/tokens/:accountAddress' component={Tokens} />
```

Comment:
Would the following routes make sense?

`/tokens/:accountAddress`
`/whitelist/:accountAddress`
`/send/:token/from/:accountAddress`

So that Tokens don't have so many routes.